lib/auth/auth.js

```diff
@@ -87,9 +87,10 @@ export default class Auth {
   }
 
   mounted () {
-
-    // handle token expiration, if axios credentials is eneabled
-    this.watchTokenExpiration()
+    // handle token expiration    
+    if (this.options.refreshToken.autoRefresh) {
```

Comment:
@breakingrobot yes, it could be. But it just ignore when some strategy doest not provide any refresh token. But, if you find that is mandatory to approve the PR, i can do it. But it does not add any advantage.

Reply:
@breakingrobot Done!